### ISSUE TITLE
[Feature Request] Some suggestions for the project

### ISSUE THREAD

**Issue by frapples** [oliverschwendener/ueli#295]
Hi, thank you for creating such an great program, it has greatly improved my productivity.
Here are some suggestions:
1. Currently you can only use the up and down keys to move through the candidate list.But I'm more accustomed to using ctrl-j and ctrl-j. It would be better if there is such a configuration item.
2. One of the advantages of similar projects like Cerebro, Zazu, Hain is to provide custom extensions, Unfortunately, these projects are no longer active. In this way, you only need to maintain a small Ueli core, and the rich external functions can be completed by the community and hobbyists writing plugins.
Thank you :)

**Comment by ToXIc-Dev** [comment]
I agree, some documentation on how to create a plugin and an easy way to install them would be great.

**Comment by oliverschwendener** [comment]
1. I see that this make sense, but I have no plans to implement this. If anyone wants to implement this, feel free to reopen.
2. This is already the case as anyone can build and extension and contribute it via GitHub pull requests.
